webapp/src/js/components/containers/GenomeBrowserWithActions.js

```diff
@@ -215,7 +215,7 @@ let GenomeBrowserWithActions = React.createClass({
                   title={sidebar ? 'Expand' : 'Sidebar'}
             />
           </div>
-          <GenomeBrowser setProps={setProps} sideWidth={150} {...subProps} />
+          <GenomeBrowser ref={(ref) => this.genomeBrowser = ref} setProps={setProps} sideWidth={150} {...subProps} />
```

Comment:
No, sorry, that's an artefact from when I falsely assumed we'd be downloading all of the Genome Browser data.
I'll remove it.